content/concepts/configuration.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Configuration
+---
+
+You may have noticed that few webpack configurations look exactly alike. This is because **webpack's configuration file is a JavaScript file that exports an object.** This object, is then parsed by webpack based upon its defined properties.
+
+## Examples
```

Comment:
Maybe drop this header. If you do that, you can push the level of following headers down by one so you get `##` for them.